data/i18n/en/en.toml

```diff
@@ -1,5 +1,13 @@
 # i18n strings for the English (main) site.
 # NOTE: Please keep the entries in alphabetical order when editing
+
+# Avoid using conjunction_1.
+# Must match the context in layouts/shortcodes/release-data.html
+# Appears on https://kubernetes.io/releases/
+# For example the "and" in "Complete 1.25 Schedule and Changelog"
+[conjunction_1]
```

Comment:
I suppose really we should sort this after the community strings.

Reply:
> I suppose really we should sort this after the community strings.

You're right, @sftim. I've moved it here: https://github.com/kubernetes/website/pull/36639